### ISSUE TITLE
How can I stop Back In Time backup?

### ISSUE THREAD

**Issue by sudhirkhanger** [bit-team/backintime#591]
Hi,

Once I have started a back I can't find a way to stop it. Is there? There's an exit button which closes the window and then the process continues in system tray. There is no button to stop in system tray applet.

![screenshot_20160624_152317](https://cloud.githubusercontent.com/assets/841656/16336085/4e27f310-3a28-11e6-93ae-96cebcc168ea.png)

Regards,
Sudhir.

**Comment by simgunz** [comment]
You can't. See issue #474. 

A bruteforce approach is to kill all the processes involving backintime for now (maybe there is a better way).

    ps aux | grep backintime

**Comment by Germar** [comment]
I'm working on it. For now you need to kill the processes manually like simgunz said. Best practice is to kill the `python3 backintime.py` process first and than the `rsync` processes.

**Comment by ketsa** [comment]
Still no kill switch ?

**Comment by XPanniX** [comment]
> A bruteforce approach is to kill all the processes involving backintime for now (maybe there is a better way).

Here is a copy & paste command from the [german ubuntuusers wiki article about backintime](https://wiki.ubuntuusers.de/Back_In_Time/#Backups-pausieren-oder-abbrechen) to kill all backintime processes:

```bash
ps auxfww|grep backintime|awk '{print $2}'|xargs sudo kill -9 
```

**Comment by colinl** [comment]
I believe there is a slight risk with that, namely that one of the PIDs returned will be that of the grep command, however by the time the kill is executed that process will already be dead and it is conceivable that the PID will have been re-used for another process.  The result would be to kill that new process.

**Comment by Germar** [comment]
This was implemented with current version 1.2.0

**Comment by hebime** [comment]
@Germar If I kill the processes from the CLI, do I have to cleanup what was written to disk? Will the partial backup affect future backups?

I ask because I'm wondering if there's a safe way to stop backintime from a UPS shutdown script.
